doc/integrations/compatibility.md

```diff
@@ -70,20 +90,24 @@ These table reflects the compatibility status of JupySQL `>=0.7`
 
 - Running queries with `%%sql` ✅
 - CTEs with `%%sql --save NAME` ✅
-- Plotting with `%%sqlplot boxplot` ❌
-- Plotting with `%%sqlplot histogram` ❌
-- Plotting with `ggplot` API ❌
+- Plotting with `%%sqlplot boxplot` ✅
+- Plotting with `%%sqlplot bar` ✅
+- Plotting with `%%sqlplot pie` ✅
+- Plotting with `%%sqlplot histogram` ✅
```

Comment:
Getting error on `taxi` dataset for histogram:

<img width="1172" alt="Screenshot 2023-08-04 at 3 27 43 PM" src="https://github.com/ploomber/jupysql/assets/29096135/93b93c34-b5ef-4ba8-83cf-cf0366778b52">

Reply:
fixed